src/main/java/de/wejago/hichi2influx/factory/JsonSubscriberFactory.java

```diff
@@ -0,0 +1,8 @@
+package de.wejago.hichi2influx.factory;
+
+import de.wejago.hichi2influx.config.Device;
+import de.wejago.hichi2influx.service.JsonSubscriber;
+
+public interface JsonSubscriberFactory {
```

Comment:
Only use Interfaces when you have MORE than one implementation, otherwise it's useless in 95% of the cases.